### PR TITLE
Fix tasks

### PR DESCRIPTION
**Pull request by IrinaBukley23**
fix tasks in columns
не понятно странное поведение модального окна при удалении: если нажать на удаление любой не последней колонки, то удаление отрабатывает несколько раз, если удалять последнюю добавленную то все норм

### CODE REVIEW

**Review by sunnyfur — APPROVED** [review]

**Comment by katerinafedotova** [comment]
the problem is in the `column.tsx` file in the `handleRemove` function. When you open the dialog, you set `open` to `true`, but on Remove you do not set it back to `false.` So, this should solve the issue
```
const handleRemove = () => {
        dispatch(setRemoveColumn(columnId))
        setOpen(false);
    };
```

File: my-app/src/Components/task/task.tsx [hunk 1]
```diff
@@ -27,44 +27,44 @@ const Task = (props: IProps) => {
...
     const handleSaveTitle = () => {
       setIsEditTitle(false);
       dispatch(editTaskTitle(taskId, correctedTitle));
     };
 
     const handleSaveDescr = () => {
-        setIsEditDescr(false);
-        dispatch(editTaskDescr(taskId, correctedDescr));
-      };
+      setIsEditDescr(false);
```
Comment (katerinafedotova) [inline my-app/src/Components/task/task.tsx:47]: Looks like there is a problem with formatting, it works differently for some of you. That's why we have these spaces here, please check

File: my-app/src/Components/taskForm/taskForm.tsx [hunk 2]
```diff
@@ -5,25 +5,28 @@ import { AnyAction } from 'redux';
...
+  const myId = nextId();
 
   const handleChange = (
     e: React.ChangeEvent<HTMLInputElement>,
     callback: (value: string) => AnyAction,
+    foo: (value: boolean) => void
   ) => {
-    (e.target.value.length < 3) ? setIsError(true) : setIsError(false);
-      (e.target.value.length >= 3 && e) ? setIsValidate(false) : setIsValidate(true);
+    (e.target.value.length < 3) ? foo(true) : foo(false);
```
Comment (katerinafedotova) [inline my-app/src/Components/taskForm/taskForm.tsx:28]: `const minNumberOfLetters = 3;`

Comment (katerinafedotova) [reply my-app/src/Components/taskForm/taskForm.tsx:28]: please some meaningful name for `foo`.

File: my-app/src/pages/boardPage/boardPage.tsx [hunk 3]
```diff
@@ -24,15 +26,15 @@ const BoardPage = () => {
       (e.target.value.length >= 3 && e) ? setIsValidate(false) : setIsValidate(true);
```
Comment (katerinafedotova) [inline my-app/src/pages/boardPage/boardPage.tsx:26]: also here put `3` into variable. And make it `e && e.target.value.length >= 3`

File: my-app/src/store/actions/actionCreators.ts [hunk 4]
```diff
@@ -49,6 +54,11 @@ export const setTaskList = (taskList: TaskItemType[]) => ({
   payload: taskList,
 })
 
+export const setCurrentColumnId = (currentColumnId: string) => ({
+  type: Actions.SET_CURRENTCOLUMNID,
```
Comment (katerinafedotova) [inline my-app/src/store/actions/actionCreators.ts:58]: separate words in  actions with underscore: `SET_CURRENT_COLUMN_ID`

File: my-app/src/types/enum.ts [hunk 5]
```diff
@@ -4,10 +4,12 @@ export enum Actions {
   SET_COLUMNLIST = 'SET_COLUMNLIST',
   EDIT_COLUMNTITLE = 'EDIT_COLUMNTITLE',
   REMOVE_COLUMN = 'REMOVE_COLUMN',
+  SET_CURRENTID = 'SET_CURRENTID',
   SET_TASKID = 'SET_TASKID',
   SET_TASKTITLE = 'SET_TASKTITLE',
```
Comment (katerinafedotova) [inline my-app/src/types/enum.ts:9]: Also for other actions please use `_` in between the words

File: my-app/src/Components/UI/OpenDialog.tsx [hunk 6]
```diff
@@ -28,7 +28,7 @@ interface PropsDialogModal {
           {'Создать задачу?'}
```
Comment (katerinafedotova) [inline my-app/src/Components/UI/OpenDialog.tsx:28]: for the name of folders. Use lowercase everywhere: Components => components, UI => ui

**Pull request merged**